templates/es6/www

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/**
+ * Entrypoint to application for es6 support
+ */
+ 
+require("babel-core/register");
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app').default;
```

Comment:
Hey, wow, I must have missed this, sorry. That's a great suggestion, it doesn't make generating harder at all. The one thing that I'm not sure about is it would cause projects generated with `express --es6` to have a slightly different directory structure. So, were we to take that route, then the base js (AKA ES5) option should look the same.

This then leads to a case where some of the simplicity is gone. Currently, when you generate a project, the only initial js file in your project's top level directory is `app.js`, which is part of the appeal. That said, putting `server.js` in there as well isn't wrong, and in fact I think it's more clear (i.e. `server.js` is where you go to make changes to the actual server and `app.js` is where you go to make changes to your app), but it would be a small departure. 

Any way, sorry for not seeing any of this. I made and pushed the changes. Nice idea!